src/authentication/footer/footer.tsx

```diff
@@ -15,10 +21,12 @@ export class Footer extends React.Component {
           <Link to='/login'>Log in</Link>
         </div>
 
-        <div>
-          <span>Dont have an invite code? </span>
-          <a href='https://www.zine.live/#/portal/signup'>Subscribe for updates</a>
-        </div>
+        {this.props.stage === RegistrationStage.ValidateInvite && (
+          <div>
+            <span>Dont have an invite code? </span>
+            <a href='https://www.zine.live/#/portal/signup'>Subscribe for updates</a>
+          </div>
+        )}
```

Comment:
why not have this in the invite component itself, if we're showing it only there? 🤔

Reply:
This area of the footer code will be updated in the follow up screens, this is a simple first iteration of this component 🙂 

Also, think it's better to keep anything footer related within the footer component so its easier to follow - as well as grouping css styles in one container rather than multiple areas.